src/collective/cover/tiles/edit.py

```diff
@@ -57,8 +58,13 @@ def handleSave(self, action):
         # fields that weren't sent with the form
         dataManager = ITileDataManager(tile)
         old_data = dataManager.get()
-        for item in data:
-            old_data[item] = data[item]
+
+        form.applyChanges(self, old_data, data)
+        for group in self.groups:
+            form.applyChanges(group, old_data, data)
+
+        current_data = self.getContent()
+        current_data.update(old_data)
```

Comment:
@cleberjsantos why is this necessary?

Reply:
@cleberjsantos It's working without it. So I removed that code. Feel free to return it if needed.